source/partials/drupal-8-convert-to-composer.md

```diff
@@ -43,23 +43,23 @@ This process involves significant changes to the codebase. We recommend you to d
   git commit -m "Pull in configuration from default branch"
   ```
 
-1. Check for `pantheon.yml` settings you need to preserve by comparing your old codebase's `pantheon.yml` to the new `pantheon.upstream.yml`:
+1. Compare the old codebase's `pantheon.yml` to the new `pantheon.upstream.yml`:
 
   ```bash{promptUser:user}
   git diff master:pantheon.yml pantheon.upstream.yml
   ```
 
   Press `q` on your keyboard to exit the diff display.
 
-   - If there are settings from `pantheon.yml` (shown with a `-` in the diff output), consider copying over your old `pantheon.yml` to preserve these settings:
+1. Copy the old `pantheon.yml` to preserve settings:
 
-     ```bash{promptUser:user}
-     git checkout master pantheon.yml
-     git add pantheon.yml
-     git commit -m 'Copy my pantheon.yml'
-     ```
+  ```bash{promptUser:user}
+  git checkout master pantheon.yml
+  git add pantheon.yml
+  git commit -m 'Copy my pantheon.yml'
+  ```
 
-   If you prefer to keep the value for `database` from `pantheon.upstream.yml`, remove it from `pantheon.yml`.
+ If you prefer to keep the value for `database` from `pantheon.upstream.yml`, remove it from `pantheon.yml`.
```

Comment:
We might more generally say that any value that is not needed from pantheon.yml can / should be removed. At a minimum, though, they should have a file and it should have `api_version: 1` in it.

Reply:
I modified [these lines to remind/reflect this](https://github.com/pantheon-systems/documentation/pull/6420/commits/8dee21155f7f3f1c2c8671d2b5052df633e44361#diff-14d96ef63a5b97fe59416ff30594a9b98d422d0fda45e712d0e2152c39451870R62-R64)
+ moving to Docs copy review

Reply:
Looks great, thanks.